packages/sources/coinmetrics-test/src/endpoint/price.ts

```diff
@@ -29,5 +48,13 @@ export const endpoint = new CryptoPriceEndpoint<AssetMetricsEndpointTypes>({
   aliases: ['price-ws'],
   transportRoutes,
   defaultTransport: 'http',
-  inputParameters: priceInputParameters,
+  inputParameters,
```

Comment:
Since we are doing validation in `customInputValidation` i think we can use `priceEndpointInputParameters` from framework and get rid of input params above. Can do the commit

Reply:
I changed it

Reply:
It may be cleaner to keep the input parameters with `options: Object.values(VALID_QUOTES),` and remove the `customInputValidation`. I'll leave it up to @alejoberardino to be the tie breaker but I'm good to approve after this.

Reply:
There were conflicting types with the type: string and options, options should be there for enum and the price endpoint expects string